hera_pspec/pspecdata.py

```diff
@@ -727,42 +765,88 @@ def scalar(self, stokes='I', taper='none', little_h=True, num_steps=2000):
                 numerical integral
                 Default: 10000
 
+        spw_select : tuple, Example: (200, 300)
```

Comment:
Maybe `spw_range` instead of `spw_select`?

Reply:
good idea